Kitodo-API/src/main/java/org/kitodo/api/dataeditor/rulesetmanagement/SimpleMetadataViewInterface.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.api.dataeditor.rulesetmanagement;
+
+import java.util.Collection;
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * Provides an interface for the meta-data key view service. The meta-data key
+ * view service provides a filtered view on meta-data keys.
+ */
+public interface SimpleMetadataViewInterface extends MetadataViewInterface {
+    /**
+     * Maps a boolean value to a meta-data value.
+     *
+     * @param value
+     *            boolean input value
+     * @return value to save as a meta-data entry. If absent, delete the
+     *         meta-data entry.
+     */
```

Comment:
The method returns the first key when true. What us rhe "mapping" part here?

Reply:
The two boolean values true and false are mapped to the two states in the METS, true = the first value, false = no value.